### ISSUE TITLE
Add a new organization to ROR: Ribbon Communications

### ISSUE THREAD

**Issue by mariagould** [ror-community/ror-updates#207]
Summary of request: Add a new organization to ROR

TBD depending on #183 and #186

**Comment by mariagould** [comment]
Ribbon Communications (US) now exists in ROR: https://ror.org/02mzerg05. I believe this happened via a renaming by GRID as this ID was previously associated with Genband, which merged into Ribbon in 2017. 

The entry in ROR for Ribbon Communications (US) has a child ID for Ribbon Communications (Israel): https://ror.org/00pfa9y72

I am closing this issue as it is no longer necessary to create a new entry for Ribbon. When we do wholesale work across the registry to handle records that were made inactive by GRID, we can address the question of how to represent historical relationships and historical names in the metadata.
